instance/models/mixins/openedx_storage.py

```diff
@@ -61,7 +61,10 @@ def get_storage_settings(self):
         Only needed when not using ephemeral databases
```

Comment:
@replaceafill  nit: Could you remove this comment?  It doesn't apply anymore, since you've added `ephemeral.yml`.